3-extra/1-radio-stations.js

```diff
@@ -26,6 +41,17 @@
  */
 // `getStations` goes here
 
+
+function getStations(allFrequencies) {
+  let arrayOfRadioStations = []
+  allFrequencies = getAllFrequencies()
```

Comment:
getStations function is getting allFrequencies as a parameter but in this line, allFrequencies get assigned the return value of the getAllFrequencies function.